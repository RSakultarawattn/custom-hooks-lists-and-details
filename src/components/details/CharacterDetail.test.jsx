import React from 'react';
import { render, cleanup } from '@testing-library/react';
import CharacterDetail from './CharacterDetail';

describe('display character details', () => {
  afterEach(() => cleanup());
  it('renders character details', () => {
    const { asFragment } = render(<CharacterDetail
      image=""
      name=""
      status=""
      species=""
    />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
