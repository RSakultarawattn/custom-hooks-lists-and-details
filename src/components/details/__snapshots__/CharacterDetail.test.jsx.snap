// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display character details renders character details 1`] = `
<DocumentFragment>
  <figure>
    <img
      alt=""
      src=""
    />
    <figcaption>
      <p />
      <p />
      <p />
    </figcaption>
  </figure>
</DocumentFragment>
`;
